clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/bitbucket/BitbucketArtifactAccount.java

```diff
@@ -29,20 +30,29 @@
 
 @NonnullByDefault
 @Value
-public class BitbucketArtifactAccount implements ArtifactAccount, BasicAuth {
-  private final String name;
-  private final Optional<String> username;
-  private final Optional<String> password;
-  private final Optional<String> usernamePasswordFile;
+public class BitbucketArtifactAccount implements ArtifactAccount, BasicAuth, TokenAuth {
+  String name;
```

Comment:
I believe these can still be private

Reply:
@Value already marks non-static fields final.
@Value already marks non-static, package-local fields private.

Reply:
![Screen Shot 2022-01-12 at 14 08 22](https://user-images.githubusercontent.com/9583719/149205779-5c31515d-fc8f-466b-8fe9-2facd067c2da.png)